src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,44 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconProps } from "@components/Icon/Icon";
+import { Dispatch, ReactElement, SetStateAction } from "react";
+import css from "./Slider.module.css";
+
+export interface IconItem {
+    icon: ReactElement<IconProps>;
+    name: string;
+}
+
+export interface SliderProps {
+    items: string[] | IconItem[];
+    activeItem: string;
+    onChange: Dispatch<SetStateAction<string>>;
+}
+
+const isIconItem = (item: string | IconItem): item is IconItem => (item as IconItem).icon !== undefined;
+
+export default function Slider({ items, activeItem: active, onChange }: SliderProps): ReactElement<SliderProps> {
```

Comment:
@SamuelAlev not sure if naming this component `Slider` is somehow misleading, wdyt? (if frontify always names these kind of component slider i'm fine with it 🙂 )